light-client/src/consensus/rpc/portal_rpc.rs

```diff
@@ -0,0 +1,316 @@
+use crate::consensus::rpc::ConsensusRpc;
+use async_trait::async_trait;
+use ethportal_api::consensus::light_client::bootstrap::LightClientBootstrap;
+use ethportal_api::light_client::bootstrap::LightClientBootstrapCapella;
+use ethportal_api::light_client::finality_update::LightClientFinalityUpdateCapella;
+use ethportal_api::light_client::optimistic_update::LightClientOptimisticUpdateCapella;
+use ethportal_api::light_client::update::LightClientUpdateCapella;
+use ethportal_api::types::consensus::light_client::finality_update::LightClientFinalityUpdate;
+use ethportal_api::types::consensus::light_client::optimistic_update::LightClientOptimisticUpdate;
+use ethportal_api::types::consensus::light_client::update::LightClientUpdate;
+use ethportal_api::types::content_key::beacon::LightClientFinalityUpdateKey;
+use ethportal_api::types::content_key::beacon::LightClientOptimisticUpdateKey;
+use ethportal_api::{
+    BeaconContentKey, BeaconContentValue, ContentValue, LightClientBootstrapKey,
+    LightClientUpdatesByRangeKey,
+};
+use eyre::eyre;
+use futures::channel::oneshot;
+use portalnet::overlay_service::OverlayCommand;
+use std::time::{SystemTime, UNIX_EPOCH};
+use tokio::sync::mpsc::UnboundedSender;
```

Comment:
**nitpick** can you reorder the imports so we have a group of `std`, followed by a group of external libraries, followed by a group of internal imports (and in the other `light-client` libraries)

Reply:
[rustfmt](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#imports_granularity) has such type of formats, it will be better if we start using them and keep this consistent through the repo, instead of manually changing those files.